### PR TITLE
Adds support for float env types

### PR DESCRIPTION
**Pull request by johnistan**
Add support for float env vars. 

**Here is a cute animal for your troubles...**
image

### CODE REVIEW

**Comment by pipermerriam** [comment]
:shipit:

File: excavator/utils.py [hunk 1]
```diff
@@ -69,6 +69,34 @@ def env_int(name, required=False, default=empty):
     return int(value)
 
 
+def env_float(name, required=False, default=empty):
+    """Pulls an environment variable out of the environment and casts it to an
+    integer. If the name is not present in the environment and no default is
```
Comment (Thanavath) [inline excavator/utils.py]: float

**Comment by Thanavath** [comment]
:+1:

**Comment by jofusa** [comment]
on a side note. we need to get lunch sometime

**Comment by Thanavath** [comment]
agreed. Jaipur or Rueben's

**Comment by jofusa** [comment]
reubens today 12:30?

**Comment by Thanavath** [comment]
I'm working from home today. :disappointed:

**Comment by jofusa** [comment]
can i get a bump on this

**Comment by pipermerriam** [comment]
released on pypi as 1.5

**Pull request merged**